rotkehlchen/chain/gnosis/modules/xdai_bridge/decoder.py

```diff
@@ -0,0 +1,104 @@
+from typing import TYPE_CHECKING, Any
+
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.chain.evm.decoding.constants import XDAI_BRIDGE_CPT_DETAILS
+from rotkehlchen.chain.evm.decoding.interfaces import DecoderInterface
+from rotkehlchen.chain.evm.decoding.structures import (
+    DEFAULT_DECODING_OUTPUT,
+    DecoderContext,
+    DecodingOutput,
+)
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails, EventCategory
+from rotkehlchen.chain.evm.decoding.utils import bridge_match_transfer, bridge_prepare_data
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.chain.gnosis.constants import CPT_XDAI_BRIDGE
+from rotkehlchen.constants.assets import A_XDAI
+from rotkehlchen.types import ChainID, ChecksumEvmAddress, DecoderEventMappingType
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+BRIDGE_ADDRESS = string_to_evm_address('0x7301CFA0e1756B71869E93d4e4Dca5c7d0eb0AA6')
+BRIDGE_TOKEN = b'\x12vP\xbc\xfb\x0b\xa0\x17@\x1a\xbeI1E:@Q@\xa8\xfd6\xfe\xceg\xba\xe2\xdb\x17M?\xddc'  # noqa: E501
```

Comment:
in the bridge for what I saw you can only bridge xdai so maybe to avoid confusions is better just

```suggestion
BRIDGE_XDAI= b'\x12vP\xbc\xfb\x0b\xa0\x17@\x1a\xbeI1E:@Q@\xa8\xfd6\xfe\xceg\xba\xe2\xdb\x17M?\xddc'  # noqa: E501
```